encoding/dot/decode_test.go

```diff
@@ -84,142 +83,124 @@ const undirected = `graph H {
 // round-trip encoding and decoding of DOT graphs with nodes and edges
 // containing DOT attributes.
 
-// DirectedGraph extends simple.DirectedGraph to add NewNode and NewEdge
+// dotDirectedGraph extends simple.DirectedGraph to add NewNode and NewEdge
```

Comment:
In test code, I usually just call the implementations things like `directedGraph`, `node` and `edge`. No need to change these though.